### ISSUE TITLE
Porting to a new map

### ISSUE THREAD

**Issue by OdyseusBR** [IrregularElements/A3-DynamicReconOps#1]
Are there any documentation on how to port this mission to a new map?

**Comment by interruptinuse** [comment]
Not really! This repo was intended as a personal backup, it's undocumented, the entire PBO packing script is horrible and only works on Linux using obscure VCS tools.

This repo is structured like this:
- [`src`](https://github.com/IrregularElements/A3-DynamicReconOps/tree/master/src) is the original codebase shared between all DRO uploads. It's independent of the map, but I extracted it from the Altis version on Steam. Note that this might be a violation of the content license, so I'm not uploading this on Steam.
- In [`maps`](https://github.com/IrregularElements/A3-DynamicReconOps/tree/master/maps), each subdirectory is named after the corresponding `worldName`/`CfgWorlds` entry. This is also the part of the PBO that comes before the extension (mission PBOs are named `MISSION.WORLDNAME.pbo`). Basically, for each map, the packing script copies `src`, then copies the corresponding `maps` files on top, then applies the `patches`. I basically committed these by hand for each terrain I wanted to play, so I think this list of maps is incomplete.

Porting to a new terrain shouldn't be too involved. I *think* the only things in the SQM file that matter are the Game Logic module, the players, and the briefing base you spawn in. I would try the following steps:

1) Check if the Workshop has DRO for the terrain you want already. If yes, problem solved. If not, continue.

2) Find out your terrain's `worldName`.

2) Unpack the Altis PBO from the [latest release](https://github.com/IrregularElements/A3-DynamicReconOps/releases/latest) with [PBO Manager](https://github.com/winseros/pboman3) or [armake2](https://github.com/KoffeinFlummi/armake2). Copy the unpacked directory to your Arma 3 user profile's `MPMissions`. Rename the "extension" part to your `worldName` and replace spaces with `%20`. You should end up with e.g. `Dynamic%20Recon%20Ops%20-%20Aliabad%20Region.MCN_Aliabad`. Instead of this repo, you can also use the Altis PBO from Steam Workshop without the patches.

3) Open this mission in Arma 3 editor, keep the Game Logic module, move the HQ composition and the player team formation somewhere reasonable. Optionally, replace `images\recon_image.jpg` with map-appropriate art.

4) Save. Host the game and verify that the scenario works.
